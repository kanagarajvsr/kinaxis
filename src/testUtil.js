
import { BrowserRouter } from 'react-router-dom'
import { render,screen,fireEvent } from '@testing-library/react';
 
export const renderWithRouter = (ui, {route = '/'} = {}) => {
    window.history.pushState({}, 'Test page', route);
    const renders = render(ui, {wrapper: BrowserRouter})
    return { 
        screen,
        fireEvent,
        ...renders
    }
  }