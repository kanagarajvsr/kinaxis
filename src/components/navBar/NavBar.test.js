import NavBar from './NavBar';
import {renderWithRouter} from '../../testUtil';
 
describe('NavBar render', () => {
 
  test('Update Snapshot', () => { 
      const utils =  renderWithRouter(<NavBar />)
      const {asFragment}=utils;
      expect(asFragment()).toMatchSnapshot()
    })
  
  });
   


