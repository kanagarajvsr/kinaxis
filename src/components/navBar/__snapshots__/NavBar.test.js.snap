// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar render Update Snapshot 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <div
      class="wrapper"
    >
      <div
        class="search"
      >
        <input
          placeholder="Search..."
          type="text"
        />
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="SearchOutlinedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
      </div>
      <div
        class="items"
      >
        <div
          class="item"
        >
          <img
            alt=""
            class="avatar"
            src="https://images.pexels.com/photos/941693/pexels-photo-941693.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
